R/reorder.R

```diff
@@ -134,10 +134,23 @@ fct_inorder <- function(f, ordered = NA) {
 
 #' @export
 #' @rdname fct_inorder
-fct_infreq <- function(f, ordered = NA) {
+#' @param weight Optional vector of numeric values to weight by.
+fct_infreq <- function(f, weight = NULL, ordered = NA) {
```

Comment:
Is `weight` the right name?

Reply:
`fct_lump()` and the other functions in that family use `w` FWIW

I don't think there are any other weighted forcats functions to compare with.